Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,32 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            // stores the biographical information of the user
+            Console.Write("Input your first name and press enter: ");
+            string firstName = Console.ReadLine();
```

Comment:
Be consistent with what type you use. In C#, you can either use ``var`` (like you did in other places) to let compiler guess the type or specify it explicitly (like you did here)